ors-engine/src/test/java/org/heigit/ors/routing/graphhopper/extensions/manage/ORSGraphFileManagerTest.java

```diff
@@ -95,6 +95,26 @@ void setupORSGraphManager(String hash, EngineConfig engineConfig) {
         orsGraphRepoManager.setFileManager(orsGraphFileManager);
     }
 
+    void setupORSGraphManager(String hash, String customGraphFolder) {
```

Comment:
much redundancy to the previous method - why not e.g. add a method 
```suggestion
    void setupORSGraphManager(String hash, EngineConfig engineConfig, String customGraphFolder)
```

Reply:
I refactored the methods a little bit